### PR TITLE
Major dependency update

### PR DESCRIPTION
**Pull request by axelstudios**
#### What's this PR do?
- **Updates nearly all Python dependencies (except Django, and ones that rely on newer versions of Django/Python)**
- Adds a `manage.py flush_db` command that wipes all database contents (and restarts table sequences) except for static tables (EEEJ and Uniformat)
- Removes vestigial `django-autoslug` dependency
- Removes unused code/files
- Removes deprecated `NullBooleanField`
- Fixes sphinx spelling issues
- Fixes sphinx-build on Windows
- Updates tests
- Fixed the following errors when visiting the Swagger page
    - `<class 'seed.views.v3.sensors.SensorViewSet'> is not compatible with schema generation`
    - `<class 'seed.views.v3.tax_lot_properties.TaxLotPropertyViewSet'> is not compatible with schema generation`
- Fixes new Ruff issues
- Fixes typos
- Improves `npm test` to support both port 80 and 8000

#### How should this be manually tested?
Monkey test, tests should pass
For local testing, you'll (obviously) need to run `pip install -r requirements/test.txt`

### CODE REVIEW

File: seed/views/v3/sensors.py [hunk 1]
```diff
@@ -13,6 +13,9 @@
 
 
 class SensorViewSet(generics.GenericAPIView, viewsets.ViewSet, OrgMixin, ProfileIdMixin):
+    # For the Swagger page, GenericAPIView asserts a value exists for `queryset`
+    queryset = Sensor.objects.none()
```
Comment (haneslinger) [inline seed/views/v3/sensors.py:17]: why `none` and not `all`? I see mostly `all`s in the existing codebase.

File: seed/tests/test_analysis_pipelines.py [hunk 2]
```diff
@@ -1076,7 +1076,7 @@ def test_get_location(self):
         location, status = _get_location(self.property_view)
         self.assertEqual(status, "success")
         self.assertTrue(location is not None)
-        self.assertEqual(location, "730 Garcia Street, Boring, Oregon, 97080")
+        self.assertEqual(location, "1070 Hunt Street, Boring, Oregon, 97015")
```
Comment (haneslinger) [inline seed/tests/test_analysis_pipelines.py:1079]: just curious, why?

Comment (axelstudios) [reply seed/tests/test_analysis_pipelines.py:1079]: Pretty sure this one comes from bumping Faker from `v0.9.3` to `v29.0.0`. Somewhere along the way they changed their algorithm for generating addresses and such

**Review by kflemin — APPROVED** [review]
works on python 3.9.13!

**Pull request merged**